Voycar.Api.Web/Generic/Repository/Repository.cs

```diff
@@ -6,8 +6,8 @@ namespace Voycar.Api.Web.Generic.Repository;
 
 public class Repository<TEntity> : IRepository<TEntity> where TEntity : Entity
 {
-    private readonly VoycarDbContext _context;
-    private readonly DbSet<TEntity> dbSet;
+    protected readonly VoycarDbContext _context;
+    protected readonly DbSet<TEntity> dbSet;
```

Comment:
War zwar vorher schon so, aber eigentlich würde ich `dbSet` gerne in `DbSet` umbenennen. Oder haben wir einen Grund das kleinzuschreiben?

Reply:
Weiß ich nicht, weiß @Kenoalpe das? Kann ich sonst gerne ändern.

Reply:
Klassenattribute sind PascalCase soweit ich weiß.

Reply:
[US 135](https://tree.taiga.io/project/julius-boettger-voycar/us/135)